Added hijack warning if transponder is set to 7500

OrangeTiger opened the pull request.

MortaX1 committed:
Added hijack warning if transponder is set to 7500

FGDATA commented:
This looks like a good pull request. I suggest we created the pull request against yurik's. So FGMEMBERS brings this after accepted in yurik's repo.

How does it sound?

IH-COL

JWocky commented:
Okay, why do I have now this picture of hijacker-walkers in my mind? Maybe connected to the failure system (Mean Time between hijackings property?)

ghost commented:
yeah alright, I just made this simple patch to figure out how git works...

MortaX1 committed:
Added a mute switchn for 2500kg fuel alarm sound

The switch with the red cover is located on the overhead panel of the
engineers panel. If it's in the up position it is activated and mutes the
alarm sound for the "below 2500kg fuel" alarm. The warning light will keep
blinking.
